butane/build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    let dirs = [".butane/", "../example/.butane"];
```

Comment:
Why is the build in this directory responsible for cleaning up the `.butane` directory in the example directory? It feels to me as though if cleanup is going to happen, each project should be responsible for itself.

Reply:
There is a tidy perf hit by having a `build.rs`, so I was avoiding having one for each set of tests.

But I have no problem splitting it.  And adding the docs to explain why is a good idea.